### PR TITLE
Footer refacror

### PR DESCRIPTION
**Pull request by endymion1818**
This PR updates the footer component to include:

• More consistent styling to the social links on mobile
• Zopa logo, and link to www.zopa.com

### CODE REVIEW

File: src/components/molecules/ZopaFooter/SocialLinks/SocialLinks.tsx [hunk 1]
```diff
@@ -13,6 +15,18 @@ export interface ISocialLinkData {
   label: string;
 }
 
+const SFlexRow = styled(FlexRow)`
+  @media (max-width: 768px) {
```
Comment (thegrinder) [inline src/components/molecules/ZopaFooter/SocialLinks/SocialLinks.tsx]: you can use the breakpoint value from the `constants/grid` file

File: src/components/molecules/ZopaFooter/SocialLinks/SocialLinks.tsx [hunk 2]
```diff
@@ -13,6 +15,18 @@ export interface ISocialLinkData {
...
+    justify-content: center;
+    margin-top: 32px;
+  }
+`;
+
+const SFooterLink = styled(FooterLink)`
```
Comment (thegrinder) [inline src/components/molecules/ZopaFooter/SocialLinks/SocialLinks.tsx]: if you need those extra 8px on the side, you can just do `<FlexRow gutter={24}>` (it's `16` by default)

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (thegrinder) [inline .vscode/settings.json]: this probably shouldn't be commited

Comment (endymion1818) [reply .vscode/settings.json]: 🤦‍♂  thanks Pawel I'm such an idiot!!!

File: src/components/molecules/ZopaFooter/SocialLinks/SocialLinks.tsx (hunk 1 above)
Comment (endymion1818) [reply src/components/molecules/ZopaFooter/SocialLinks/SocialLinks.tsx]: 👍

File: src/components/molecules/ZopaFooter/SocialLinks/SocialLinks.tsx [hunk 4]
```diff
@@ -32,14 +47,21 @@ const socialLinksData: ISocialLinkData[] = [
...
+  <FlexRow justify="space-between">
+    <FlexCol xs={12} s={6} m={4} l={2}>
+      <FooterLink href="https://www.zopa.com">
+        <ZopaLogo />
+      </FooterLink>
+    </FlexCol>
+    <FlexCol xs={12} s={6} m={4} l={3}>
+      <SFlexRow justify="flex-end" gutter={24}>
+        {socialLinksData.map(({ component: Component, label, href }, index) => (
+          <SFooterLink href={href} aria-label={label} key={`footer-social-link-${index}`}>
```
Comment (lluia) [inline src/components/molecules/ZopaFooter/SocialLinks/SocialLinks.tsx]: Minor ( _doesn't block merge_ 😄 )  

I believe using array indexes as `key` is not recommended, we could use the `href` as the unique value instead?
```jsx
<SFooterLink  key={`footer-social-link-${href}`}>
```

Comment (endymion1818) [reply src/components/molecules/ZopaFooter/SocialLinks/SocialLinks.tsx]: 👍  done

**Pull request merged**